app/src/main/java/com/bashkevich/androidfundamentals/model/EntityMapper.kt

```diff
@@ -1,40 +1,59 @@
 package com.bashkevich.androidfundamentals.model
 
 import com.bashkevich.androidfundamentals.model.dto.ActorDto
-import com.bashkevich.androidfundamentals.model.dto.GenreDto
 import com.bashkevich.androidfundamentals.model.dto.MovieDto
-import com.bashkevich.androidfundamentals.model.entity.Actor
-import com.bashkevich.androidfundamentals.model.entity.Genre
-import com.bashkevich.androidfundamentals.model.entity.Movie
+import com.bashkevich.androidfundamentals.model.entity.ActorEntity
+import com.bashkevich.androidfundamentals.model.entity.MovieEntity
+import com.bashkevich.androidfundamentals.model.entity.MovieWithActors
+import com.bashkevich.androidfundamentals.model.viewobject.Actor
+import com.bashkevich.androidfundamentals.model.viewobject.Movie
 
 object EntityMapper {
-    fun convertToMovieFromDto(movieDto: MovieDto): Movie {
 
-        return Movie(
-            id = movieDto.id,
+    fun toMovieEntity(movieDto: MovieDto) =
```

Comment:
Minor - I like more syntax like MovieDto.toMovieEntitiy()

Reply:
should I delete class EntityMapper and put everything in entities/dtos?

Reply:
I would not put it to the DTO but probably would have a private/internal function in repo file or around with receiver

Reply:
extension function?

Reply:
Yup